test/evergreen.yml

```diff
@@ -627,6 +627,19 @@ functions:
         ${PREPARE_TEST_ENV}
         $(pwd)/test/csuite/${test_name}/test_${test_name} ${test_args|} 2>&1
 
+  "csuite tests all":
+    command: shell.exec
+    params:
+      working_dir: "wiredtiger"
+      shell: bash
+      script: |
+        set -o errexit
+        set -o verbose
+        ${PREPARE_TEST_ENV}
+        . test/evergreen/find_cmake.sh
+        cd cmake_build
+        $CTEST -L csuite ${check_args} -j ${num_jobs} --output-on-failure 2>&1
```

Comment:
"check_args" sounds a bit confusing as it's really used for excluding a given label regex. I saw the same name used in the `make check all` func, which is more reasonable. Suggest renaming to something general e.g. "extra_args" / "exclude_args" (the former is already used in the `make check directory` func.

Reply:
I am now directly using make check directory [8a755a8](https://github.com/wiredtiger/wiredtiger/pull/9779/commits/8a755a83a73c544b194a5307752a51ff82272a4d)